lib/Elastica/Query/Fuzzy.php

```diff
@@ -57,7 +57,7 @@ public function setFieldOption($param, $value)
     {
         //Retrieve the single existing field for alteration.
         $params = $this->getParams();
-        if (count($params) < 1) {
+        if (empty($params)) {
```

Comment:
Hm, reading this again. The outcome is the same but the previous version makes much more sense to be as we want to count the number of entries in $params. It doesn't matter to me if it is count(...) == 0 or size_of() == 0 or the existing one, but not replace it by empty. I don't see the benefit.